style.css

```diff
@@ -112,13 +123,44 @@ header {
   line-height: 52px;
 }
 
-.placeholder {
+.placeholder-multistories {
+  display: flex;
+  justify-content: left;
+  margin-top: 80px;
+  margin-left: 1.5rem;
   margin-right: 1.5rem;
 }
 
-.sepration-line {
-  margin-top: 30px;
-  margin-bottom: 80px;
+.placeholder {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.recent-work {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(1, 1fr);
+}
```

Comment:
- [OPTIONAL] Kindly reduce the `margin-top` between the recent work title and the first project. Take a look and fix it.

### Your Design
![your-work-spacing](https://user-images.githubusercontent.com/26597432/220605999-7a2b9895-dd6d-4d71-acf7-735cd74eb701.png)

### Figma Design
![temp-work-area](https://user-images.githubusercontent.com/26597432/220606045-dc849e19-7cec-4ca8-9281-5a86864882eb.png)